### PR TITLE
Contributing to issue #289

### PR DESCRIPTION
**Pull request by frefell**
Update image credits (#289) for blogposts between 2015-07-23 and 2015-10-30

### CODE REVIEW

File: blog/_posts/2015-07-23-rubyherzlein.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Why Rubyherzlein
 While deciding on a name, we talked about how the Rails Girls Society is so full of hearts! Its the logo for most events and also quite often used as an emoji within the RGSoC community. As we are beginners, we thought of calling ourselves Rubyherzlein, which in German means ‘little Ruby heart’! <3<3<3
 
 <img src="/img/blog/2015/rubyherzlein.JPG" alt="Team Rubyherzlein" width="600">
-<br><font color="grey"><small><i>Rubyherzlein Team + extended Team: Erik, Duana, Hannes  (Image: Rubyherzlein Team)</i></small></font> 
+<div align="center" class="image-credits">Rubyherzlein Team + extended Team: Erik, Duana, Hannes  (Image: Rubyherzlein Team) </div>
```
Comment (alicetragedy) [inline blog/_posts/2015-07-23-rubyherzlein.md]: could you remove the `align="center"` bit from all the files? Unfortunately some of the last contributors to blog posts seem to have added that, but it doesn't belong in there. Thank you so much!

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hi @frefell, wow, thank you so much for this contribution! It's so exciting to have this long open issue be taken care of ❤️ 
Before I merge this I'd like you to change something — I've left a comment about it in one of the files. If you have any question about this, don't hesitate to ping me, I'm always happy to help.

File: blog/_posts/2015-07-23-rubyherzlein.md (hunk 1 above)
Comment (frefell) [reply blog/_posts/2015-07-23-rubyherzlein.md]: Hei, thanks for your review :) I will take care of that.

**Comment by alicetragedy** [comment]
@frefell thanks for starting to make the changes! I see there's still a couple of blog posts that haven't been updated — would you like to just ping me directly in here when you're done, so that I can review it all and merge? Thanks! 🙌

**Comment by frefell** [comment]
@alicetragedy Yes I was planning that, because I will need some more days to finish this :)

**Comment by alicetragedy** [comment]
Great! Just wanted to make sure you know you can ping me when you're done (or if you need anything). 😇

**Comment by alicetragedy** [comment]
Hey @frefell, were you able to make more changes to the posts? I'd love to merge this soon — maybe there is something I can do to help?

**Comment by frefell** [comment]
Hey, @alicetragedy! 
I was very busy last week, but managed to change everything in the blogposts between 2015-07-23 and 2015-10-30, as in the first pull request, so I think this could be merged now.

If I find time to work on the older blogposts, I will create a new pull request. :)

File: blog/_posts/2015-08-07-fanxhe-team-is-broadcasting-from-rgsoc.md [hunk 2]
```diff
@@ -185,13 +156,9 @@ Do you want to see half a day of Fanxhe Team at RGSoC ?
 <div align="center">
   <iframe width="560" height="315" src="https://www.youtube.com/embed/wWgvqLD0pWM" frameborder="0" allowfullscreen></iframe>
 </div>
+<div class="image-credits"><a href= "https://twitter.com/zazvick">By @zazvic</a> </div>
 <p align="center">
```
Comment (alicetragedy) [inline blog/_posts/2015-08-07-fanxhe-team-is-broadcasting-from-rgsoc.md]: I think this line can be removed as well :)

File: blog/_posts/2015-09-16-alster-hamburgers-code-review.md [hunk 3]
```diff
@@ -68,7 +70,7 @@ And what’s next?
 Hopefully, we’ll continue to build our coding skills! We’ve learned a lot of things during those summer months but often, the questions grow at an even faster rate…
 
 <img src="/img/blog/2015/alster-learntocode-graph.jpg" alt="a graph of our learning curve" width="600px">
-<br><font color="grey"><small><i>What learning felt like at times... Illustration: Anke</i></small></font>
+<div class="image-credits">hat learning felt like at times... Illustration: Anke </div>
```
Comment (alicetragedy) [inline blog/_posts/2015-09-16-alster-hamburgers-code-review.md]: there's a `W` missing here at the beginning of the sentence 😇

File: blog/_posts/2015-09-24-A-Tale-of-Sweet-Cocoa-with-Gems.md [hunk 4]
```diff
@@ -58,4 +59,4 @@ I just want to aim a big thank you to everyone who made these three months of im
 
 
 <img src="/img/blog/2015/CG-2ndDinnerCocoaPods.JPG" alt="Team CocoaGems">
-<br><font color="grey"><small><i> Memebers of CocoaPods getting together for dinner after AltConf. Photo by Karla</i></small></font>
+<div class="image-credits">Members of CocoaPods getting together for dinner after AltConf. Photo by Karla </div>
```
Comment (alicetragedy) [inline blog/_posts/2015-09-24-A-Tale-of-Sweet-Cocoa-with-Gems.md:62]: Thank you for fixing this typo 🙌

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Awesome, this looks great! there are a couple of things that need changing (a `<p>` tag that needs removing and a missing letter). These are all minor things, so if you can't make the changes, let me know and I will make them myself after merging. <3

**Comment by frefell** [comment]
hei again, @alicetragedy. I think I found the small errors you mentioned, and everything should be nice now :)

**Review by alicetragedy — APPROVED** [review]
Whoohoo! Awesome @frefell, looks like we're ready to ship this. 🙌

**Comment by alicetragedy** [comment]
Congratulations on your very first contribution to our website, @frefell! This was a lot of tedious work and it's sooooo helpful to have these changes, so thank you so much for taking the time to improve these posts for us. 💖 💪 💻

**Pull request merged**